src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -22,4 +22,8 @@
   public static final String NODE_EXECUTOR_ID = "NodeExecutor";
   public static final String NAME_NODE_EXECUTOR_ID = "NameNodeExecutor";
 
+  // Path to Store HDFS Binary
+  public static final String MESOSPHERE_OPT_PATH = "/opt/mesosphere/";
```

Comment:
Some slave OS setups may not allow the HDFS user to modify `/opt` directly, so they will need to use a different location. I guess they can always just rebuild the HDFS framework from scratch, but I'd like to see a config parameter for it, which can of course default to `/opt/mesosphere`